src/utils/getElement.js

```diff
@@ -16,7 +16,8 @@ export const getAsyncElement = (...params) => {
   const element = resolveValidElement(...params);
 
   if (element instanceof HTMLImageElement) return imageLoader(element);
-  if (element instanceof HTMLCanvasElement) return Promise.resolve(getElementInfo(element));
+  else if (element instanceof HTMLCanvasElement) return Promise.resolve(getElementInfo(element));
```

Comment:
You don't need to add the `else` here, since it is using early returns. This was a code style option from my part.

Reply:
I have fixed the formatting. Personally, I prefer to have the `else if` as it serves as a visual indicator that only one of the if statements should run.